packages/playground/src/dashboard/components/create_farm.vue

```diff
@@ -19,11 +19,13 @@
                 :rules="[
                   validators.required('Farm name is required.'),
                   name => validators.isAlpha('Farm name must start with an alphabet char.')(name[0]),
-                  validators.minLength('Farm name minimum length is 2 chars.', 2),
+                  validators.minLength('Farm name minimum length is 3 chars.', 3),
                   validators.maxLength('Farm name maximum length is 40 chars.', 40),
                   validators.pattern('Farm name  should not contain whitespaces.', {
                     pattern: /^[^\s]+$/,
                   }),
+
+                  validateFarmName,
```

Comment:
can we use validators.pattern like the above one ?

Reply:
and please remove the extra space in line 24 after `Farm name`

Reply:
validators.pattern checks only if pattern exists, it does not check if every character satisfies pattern.